packages/@aws-cdk/aws-lambda-nodejs/lib/bundling.ts

```diff
@@ -298,13 +299,17 @@ interface BundlingCommandOptions {
 class OsCommand {
   constructor(private readonly osPlatform: NodeJS.Platform) {}
 
-  public writeJson(filePath: string, data: any): string {
-    const stringifiedData = JSON.stringify(data);
+  public write(filePath: string, data: any): string {
```

Comment:
Can you provide further context on why this additional function is needed?

Reply:
I have created `write()` which takes a string and outputs it to a file. This is required for the `.npmrc` and `pnpm-workspace.yaml` files.

`writeJson()` receives any input, stringifies it, then calls `write()`.

This way I have avoided duplicate code.